bindings/nodejs/examples/client/03_get_address_outputs.ts

```diff
@@ -33,7 +33,7 @@ async function run() {
         console.log('Output ids: ', outputIdsResponse, '\n');
 
         const addressOutputs = await client.getOutputs(outputIdsResponse.items);
-        console.log('Address outputs: ', addressOutputs);
+        //console.log('Address outputs: ', addressOutputs);
```

Comment:
Why commented?

Reply:
my bad, was too much to see the previous logging so i disabled it